nvflare/private/fed/simulator/simulator_server.py

```diff
@@ -144,6 +145,18 @@ def _create_server_engine(self, args, snapshot_persistor):
 
     def deploy(self, args, grpc_args=None, secure_train=False):
         super(FederatedServer, self).deploy(args, grpc_args, secure_train)
+        workspace = Workspace(args.workspace, "server", args.config_folder)
+        run_manager = RunManager(
```

Comment:
2024-11-26 14:05:11,298 - SimulatorRunner - ERROR - Simulator setup error: RuntimeError: invalid workspace /home/yuantingh/FATE_Exp/nv_workspace/jobs/simulator_workspace: missing site config folder '/home/yuantingh/FATE_Exp/nv_workspace/jobs/simulator_workspace/local' or not a valid dir
2024-11-26 14:05:11,301 - root - ERROR - Traceback (most recent call last):
  File "/home/yuantingh/NVFlare/nvflare/private/fed/app/simulator/simulator_runner.py", line 247, in setup
    simulator_server, self.server = self.deployer.create_fl_server(self.args)
                                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/yuantingh/NVFlare/nvflare/private/fed/app/deployer/simulator_deployer.py", line 54, in create_fl_server
    services.deploy(args, grpc_args=simulator_server)
  File "/home/yuantingh/NVFlare/nvflare/private/fed/simulator/simulator_server.py", line 148, in deploy
    workspace = Workspace(args.workspace, "server", args.config_folder)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/yuantingh/NVFlare/nvflare/apis/workspace.py", line 72, in __init__
    raise RuntimeError(
RuntimeError: invalid workspace /home/yuantingh/FATE_Exp/nv_workspace/jobs/simulator_workspace: missing site config folder '/home/yuantingh/FATE_Exp/nv_workspace/jobs/simulator_workspace/local' or not a valid dir

Reply:
Added handle for the workspace creation.